cloud_Azure/terraform/module/storage_account.tf

```diff
@@ -3,7 +3,7 @@
 resource "azurerm_storage_account" "logs_storage_account" {
   count = length(var.resource_group_names)
   
-  name                     = "${var.prefix}flowlogstorage${tostring(count.index)}" # eg. acmeflowlogstorage0, max 27 letters
+  name                     = "logs${terraform.workspace}${tostring(count.index)}" # constrainst: Azure-wide unique, up to 24 alphanum letters
```

Comment:
I don't think the `terraform_workspace`  name (a hexdigit string) is suitable to be part of a human consumable name.

Reply:
Yes, the combination of required global uniqueness, limited name length and character choice make the problem quite hard. Using random name/phrase generator is not a bad idea. It at least makes the name pronounceable. We should give users option to provide storage account prefix and/or complete name (say via profiles.ini). Keep in mind that in relatively near future, we will also have to support Kentik managed storage accounts.

Reply:
Kentik managed storage accounts, does it mean: use already existing storage account specified in profile instead of creating new one?

Reply:
To simplify the implementation and cost considerations for customers, Kentik is provisioning a unique storage location per customer, per region.

example
Path: arn:aws:s3:::kentik-hosted-fl-us-xxxxxx-yyyyyyy-useast1

Reply:
As Ted stated above, the core is that Kentik will **optionally** provision storage accounts for customers in our own account. Customer provisioned storage accounts will likely remain to be the default. 

Detailed mechanics of who/when will provision storage accounts fro customers are not defined yet. My preferred solution would be that the ID of the storage account is returned in response to creation of the cloudexport in Kentik (or asynchronously later). So, if a user chooses to use Kentik provided storage account, the TF provisioning process would need to start by creating cloudexport in Kentik and then proceeding to provisioning the cloud side. In any case, we do not need to solve it now.

Reply:
Updating to generate names that concatenate Resource Group name and beginning of Subscription ID.
For example:
`prodresourcegroupa39991e`